docs/testing-components/unit-testing/index.md

```diff
@@ -194,36 +182,38 @@ This example looks quite silly, but let's look at it in a React context:
 
 ```js
 //original code
-const Text = (props) => {
-	return <p>{props.content}</p>;
+const Text = ({content, ...rest}) => {
+	return <p {...rest}>{content}</p>;
 }
 
 //breaking change
-const Text = (props) => {
-	return <p>{props.cont}</p>;
+const Text = ({cont, ...rest}) => {
+	return <p {...rest}>{cont}</p>;
 }
```

Comment:
Let's not change these line.

Reply:
I would prefer to keep them so it better fits the example below.